apps/studio/pages/project/[ref]/settings/auth.tsx

```diff
@@ -19,6 +20,7 @@ import type { NextPageWithLayout } from 'types'
 
 const AuthSettings: NextPageWithLayout = () => {
   const isPermissionsLoaded = usePermissionsLoaded()
+  // TODO: check if these permissions cover third party auth as well
```

Comment:
just to check - do they cover third party auth?

Reply:
Talked with @hf, confirmed that we should use these permissions for 3rd Party Auth.